### PR TITLE
feature : annotate plastic components and ESModule SDK

### PR DESCRIPTION
**Pull request by inureyes**
This PR annotates plastic components and ESModule SDK

### CODE REVIEW

File: src/plastics/lablup-shields/lablup-shields.ts [hunk 1]
```diff
@@ -22,13 +45,9 @@ export default class LablupShields extends LitElement {
   @property({type: String}) app = '';
   @property({type: String}) description = '';
   @property({type: String}) color = 'green';
-  @property({type: String}) appColor = 'grey';
+  @property({type: String}) appColor = "grey";
```
Comment (agatha197) [inline src/plastics/lablup-shields/lablup-shields.ts]: Is there a reason why you changed it to double quotation marks?

File: src/plastics/lablup-shields/lablup-shields.ts [hunk 2]
```diff
@@ -59,25 +78,26 @@ export default class LablupShields extends LitElement {
         }
       `];
   }
-
+
```
Comment (agatha197) [inline src/plastics/lablup-shields/lablup-shields.ts]: Remove these blanks please 😉

**Review by Sujin-Kim1 — CHANGES_REQUESTED** [review]
I left a few reviews. And I think it's better to erase the `shadowRoot: any` on the `lablup-shields` too!

**Comment by inureyes** [comment]
@Sujin-Kim1 Fixed with your comments.

**Review by Sujin-Kim1 — APPROVED** [review]
LGTM!

**Pull request merged**